fetch.bs

```diff
@@ -1360,6 +1394,10 @@ specified. [[!CSP]]
 `<a http-header><code>Access-Control-Expose-Headers</code></a>` header. This list is used by a
 <a>CORS filtered response</a> to determine which headers to expose.
 
+<p>A <a for=/>response</a> has an associated
+<dfn for=response id=concept-response-range-requested-flag>range requested flag</dfn>, which is
+initially unset.
```

Comment:
It'd be good to add a note here about the purpose. Also, "range-requested" perhaps?

Reply:
Also, do we need to export this or is it fine for just internal usage?

Reply:
I don't think we'll need this externally, but I guess we can export it if needed. I also filed https://github.com/tabatkins/bikeshed/issues/1216, which would make non-exported definitions more useful.

Reply:
If we don't, then don't export it. I'd rather get requests when it's needed than find out at some point that a dependency on it was taken.

Reply:
This still needs a note about the purpose and be changed to "range-requested" (note the hyphen).